### PR TITLE
FEATURE: Add link to developer/GH info in Admin dropdown

### LINKED ISSUE #3: FEATURE: Add link to developer/GH info in Admin dropdown
### Have you updated the title?

- [X] I have updated the issue's title.

### As a...

developer

### I can...

navigate to the GitHub info page

### So that...

I can easily view the GitHub branch info

### Discussion

Currently there is a page at /developer which contains developer info about the GitHub branch, SHA, etc. Please add a link to this page to the Admin dropdown in the navbar.

### Acceptance Criteria

- [ ] There is an item in the Admin dropdown called "Developer Info".
- [ ] Clicking on this item redirects to [app-link]/developer.

### Implementation Todos

Frontend:
- [ ] Add the "Developer Info" item to AppNavbar.js
- [ ] Ensure that this item routes to /developer.

Testing:
- [ ] Front end tests pass and there is adequate coverage
- [ ] Back end tests pass and there is adequate coverage

### PR DESCRIPTION
**Pull request by samanthwest**
added link when signed in as an admin that accesses developer info

This is what it looks like in the drop down menu:

Screenshot 2024-02-28 at 2 22 22 PM

And then once clicked on:

Screenshot 2024-02-28 at 2 22 57 PM

Dokku dev link:
https://proj-colefoster08-dev.dokku-08.cs.ucsb.edu/

Closes #3

### CODE REVIEW

**Review by megumi-ondo — DISMISSED** [review]
LGTM!

**Review by tallyhawley — CHANGES_REQUESTED** [review]
Please include a link to your dokku deployment with these changes so we can test them out!

File: frontend/src/main/components/Nav/AppNavbar.js [hunk 1]
```diff
@@ -36,21 +37,13 @@ export default function AppNavbar({
 
           <Navbar.Toggle />
 
-          <>
```
Comment (tallyhawley) [inline frontend/src/main/components/Nav/AppNavbar.js]: why was this removed?

Comment (samanthwest) [reply frontend/src/main/components/Nav/AppNavbar.js]: i must've done it by accident. everything should be fixed now!

**Review by tallyhawley — COMMENTED** [review]
Looks good to me :shipit:

**Review by tallyhawley — APPROVED** [review]

**Review by megumi-ondo — APPROVED** [review]
Changes LGTM
